From 2e3517026777e5a4ea0e82c7fbc9b90d8e337d9e Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-16.27-2e351702677-20220331T005325Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.27-00000000000-20220331T005325Z-avarab@gmail.com>
References: <cover-v3-00.27-00000000000-20220325T171340Z-avarab@gmail.com>
	<cover-v4-00.27-00000000000-20220331T005325Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 30 Sep 2021 18:02:05 +0200
Subject: [PATCH v4 16/27] revisions API: have release_revisions() release
 "commits"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Extend the the release_revisions() function so that it frees the
"commits" in the "struct rev_info".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 revision.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/revision.c b/revision.c
index 8fc157e9ee7..394e2dcdd90 100644
--- a/revision.c
+++ b/revision.c
@@ -2937,6 +2937,7 @@ static void release_revisions_commit_list(struct rev_info *revs)
 
 void release_revisions(struct rev_info *revs)
 {
+	release_revisions_commit_list(revs);
 	object_array_clear(&revs->pending);
 }
 
-- 
2.35.1.1557.g4e3e9cdb5e0

